apiml-security-common/src/main/java/org/zowe/apiml/security/common/config/AuthConfigurationProperties.java

```diff
@@ -84,11 +84,13 @@ public AuthConfigurationProperties() {
      * @throws AuthenticationServiceException if the z/OSMF service id is not configured
      */
     public String validatedZosmfServiceId() {
-        if ((zosmfServiceId == null) || zosmfServiceId.isEmpty()) {
-            apimlLog.log("org.zowe.apiml.security.zosmfNotFound");
-            throw new AuthenticationServiceException("The parameter 'zosmfServiceId' is not configured.");
-        }
 
+        if (provider.equalsIgnoreCase("zosmf")) {
```

Comment:
What about using the Value from AuthenticationScheme?

Reply:
Great idea, i changed it! Thanks